config/bin_version_strings.cfg

```diff
@@ -330,29 +331,37 @@ glibc;;lgpl;"GNU\ C\ Library\ development\ release\ version\ [0-9](\.[0-9]+)+?$"
 glibc;;lgpl;"GNU\ C\ Library\ \(.*\)\ stable\ release\ version\ [0-9](\.[0-9]+)+?(\.)?$";"sed -r 's/GNU\ C\ Library\ \(.*\)\ stable\ release\ version\ ([0-9](\.[0-9]+)+?)$/gnu:glibc:\1/'";
 libcurl;;mit-style;"CLIENT\ libcurl\ [0-9](\.[0-9]+)+?";"sed -r 's/CLIENT\ libcurl\ ([0-9](\.[0-9]+)+?).*/libcurl:\1/'";
 libcurl;;mit-style;"^libcurl\/[0-9](\.[0-9]+)+?$";"sed -r 's/libcurl\/([0-9](\.[0-9]+)+?)$/libcurl:\1/'";
+libdbus;;unknown;"libdbus\ [0-9](\.[0-9]+)+?$";"sed -r 's/libdbus\ ([0-9](\.[0-9]+)+?)$/freedesktop:dbus:\1/'";
```

Comment:
can we use ^for the beginning of the line in the first identifier?

Reply:
Yes, added